client/post-editor/editor-image-selector/dropzone-icon.jsx

```diff
@@ -0,0 +1,24 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+
+import React from 'react';
+
+export default () => (
+	<svg width="55" height="42" viewBox="0 0 55 42" xmlns="http://www.w3.org/2000/svg">
+		<title>Image Selector</title>
```

Comment:
This string needs to be passed through `translate`.

Reply:
Updated!